src/Microsoft.Azure.NotificationHubs/CollectionQueryResult.cs

```diff
@@ -32,7 +32,7 @@ internal CollectionQueryResult(IEnumerable<T> results, string continuationToken)
         /// Gets the continuation token.
         /// </summary>
         /// <value>
-        /// The continuation token.
+        /// The continuation token, which is null or empty when there is no additional data available in the query.
```

Comment:
Is there unpredictability into whether it could be null in some conditions and empty in others? If we are consistent in some way (like always empty string or always null value), then it'd be great if we could remove that ambiguity from the language here. Same with the other updates.

Reply:
When we use the token in our code for queries, we check with `IsNullOrEmpty` and there is no clear documentation of when we ever get back a `null` vs `string.Empty` token from our queries. Leaving this as-is 🙂